### DISCUSSION TITLE
Getting an error with Publication Center

### DISCUSSION THREAD

**Discussion by chococirno** [oleeskild/obsidian-digital-garden#536] [Q&A]
![新标签页 - 知识库测试 - Obsidian v1 5 3 2024_1_6 22_43_06](https://github.com/oleeskild/obsidian-digital-garden/assets/155754259/5bf1c390-d780-4217-be26-3cf4cde62502)
hi,everything is normal,deployed and published successfully,but it's wrong.Please help me.

**Comment by ideastouch** [comment]
was that a try to make a static site for GitHub Pages? And was it a private or public page (site)?
